kahuna/public/stylesheets/main.css

```diff
@@ -974,7 +974,8 @@ input.related-labels__parent {
 }
 
 .related-labels__labels {
-    overflow: auto;
+    overflow-x: auto;
```

Comment:
I'm getting it mixed up - it's when you mix it with `visible`, this doesn't wokr.
:+1: